.woodpecker/test.yml

```diff
@@ -136,7 +136,7 @@ services:
     when: *when
 
   service-mysql:
-    image: mysql:5.6.27
```

Comment:
Should we maybe even switch it to `mariadb`?

Reply:
What would be the advantage? They have arm builds for older versions? 

I am not an expert in mysql/mariadb but given that MySQL 5 is EOL quite soon...

Reply:
Actually it doesn't matter too much. Some users / distros etc seem to move to mariadb as of license / philosophy reasons.